docs/tutorials/getting_started.md

```diff
@@ -61,118 +65,229 @@ directory:
+udm                                active       1  sdcore-udm-k8s                1.3/edge            35  10.152.183.168   no
+udr                                active       1  sdcore-udr-k8s                1.3/edge            31  10.152.183.96    no
+upf                                active       1  sdcore-upf-k8s                1.3/edge            64  10.152.183.126   no
+webui                              active       1  sdcore-webui-k8s              1.3/edge            23  10.152.183.128   no
 
 Unit                         Workload  Agent  Address      Ports  Message
 amf/0*                       active    idle   10.1.182.23
 ausf/0*                      active    idle   10.1.182.18
 gnbsim/0*                    active    idle   10.1.182.50
-grafana-agent-k8s/0*         blocked   idle   10.1.182.51         logging-consumer: off, grafana-cloud-config: off
-mongodb-k8s/0*               active    idle   10.1.182.35         Primary
+grafana-agent/0*             blocked   idle   10.1.182.51         logging-consumer: off, grafana-cloud-config: off
+mongodb/0*                   active    idle   10.1.182.35         Primary
 nms/0*                       active    idle   10.1.182.2
 nrf/0*                       active    idle   10.1.182.53
 nssf/0*                      active    idle   10.1.182.48
 pcf/0*                       active    idle   10.1.182.46
 router/0*                    active    idle   10.1.182.57
 self-signed-certificates/0*  active    idle   10.1.182.56
 smf/0*                       active    idle   10.1.182.27
-traefik-k8s/0*               active    idle   10.1.182.40
+traefik/0*                   active    idle   10.1.182.40
 udm/0*                       active    idle   10.1.182.52
 udr/0*                       active    idle   10.1.182.39
 upf/0*                       active    idle   10.1.182.60
 webui/0*                     active    idle   10.1.182.33
 ```
 
-## 4. Deploy the 5G simulator
+## 6. Configure the ingress
 
-Deploy the `sdcore-gnbsim-k8s` operator
+Configure Traefik to use an external hostname. To do that, edit `traefik_config` 
+in the `main.tf` file:
 
-```console
-juju deploy sdcore-gnbsim-k8s gnbsim --trust --channel=beta
 ```
-
-Integrate it to the AMF and the NMS:
-
-```console
-juju integrate gnbsim:fiveg-n2 amf:fiveg-n2
-juju integrate gnbsim:fiveg_gnb_identity nms:fiveg_gnb_identity
+:caption: main.tf
+(...)
+module "sdcore" {
+  (...)
+  traefik_config = {
+    routing_mode      = "subdomain"
+    external_hostname = "10.0.0.4.nip.io"
+  }
+  (...)
+}
+(...)
 ```
 
-## 5. Configure the ingress
-
-Configure Traefik to use an external hostname:
+Apply new configuration:
 
 ```console
-juju config traefik-k8s external_hostname=10.0.0.3.nip.io
+terraform apply -auto-approve
 ```
 
-Here, replace `10.0.0.3` with the Application IP address of the `traefik-k8s` application. You can find it by running `juju status traefik-k8s`.
+```{note}
```

Comment:
I think this should be an unconditional step before the step to edit the `main.tf` file.  Like:

* run `juju status traefik`
* note the IP address
* edit the `main.tf`

Reply:
Yeah, makes sense.
Done.